### PR TITLE
Make option body not required

### LINKED ISSUE #119: make imagePoll options optional instead of required
### Updates
Update:
After some discussions and agreements, this will be always optional and any logic needed for certain post types will be handled on the front-end side.

### Story
- according to our design the option ``body`` in the image poll type is optional and in the other two types is required. if the user left the options empty at the image poll type and tries to create the post this leads to a ``400`` error.
image

### Tasks
- [ ] make the option ``body`` in image poll type optional

### PR DESCRIPTION
**Pull request by amroakmal**
Closes #119 

- Make `option.body` to be optional/not_required.
- Make necessary changes and updates for the migration files.

### CODE REVIEW

File: examples/examples.http [hunk 1]
```diff
@@ -1,7 +1,7 @@
-@postId = 6c22e70b-33c5-406c-8cae-728b52530798
```
Comment (Sahl-A) [inline examples/examples.http]: Please try not to add changes to this file in the PR unless it is core changes.

File: src/shared/migrations/1626219362732-update_makeOptionBodyNotRequired.ts [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class updateMakeOptionBodyNotRequired1626219362732
+  implements MigrationInterface {
+  name = 'updateMakeOptionBodyNotRequired1626219362732';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    await queryRunner.query(
+      `ALTER TABLE "pickify_posts"."options" ALTER COLUMN "body" DROP NOT NULL`,
```
Comment (Sahl-A) [inline src/shared/migrations/1626219362732-update_makeOptionBodyNotRequired.ts]: Please modify `pickify_posts` with `${POSTS_SCHEMA}`.
This should be added in all file

**Review by Sahl-A — CHANGES_REQUESTED** [review]

File: src/shared/migrations/1626219362732-update_makeOptionBodyNotRequired.ts (hunk 2 above)
Comment (amroakmal) [reply src/shared/migrations/1626219362732-update_makeOptionBodyNotRequired.ts]: Done

File: examples/examples.http (hunk 1 above)
Comment (amroakmal) [reply examples/examples.http]: Sure, ok.

**Review by Sahl-A — APPROVED** [review]
LGTM

**Pull request merged**